docs/src/SUMMARY.md

```diff
@@ -23,7 +23,8 @@
   - [Recipes](45-recipes.md)
 - [Tools](50-tools.md)
   - [MBTiles Metadata](51-mbtiles-meta.md)
-  - [MBTiles Copying / Diffing](52-mbtiles-copy.md)
-  - [MBTiles Validation](53-mbtiles-validation.md)
-  - [MBTiles Schemas](54-mbtiles-schema.md)
+  - [MBTiles statistics](52-mbtiles-stats.md)
```

Comment:
we don't want to break all internet links every time we add a new docs page... we ought to figure out a way not to do that. For now, lets just add thing at the end.